anyway/infographics_utils.py

```diff
@@ -560,6 +560,39 @@ def stats_accidents_by_car_type_with_national_data(
     return out
 
 
+def get_injured_accidents_with_pedestrians(location_info, start_time, end_time):
+    if not ('yishuv_name' in location_info and 'street1_hebrew' in location_info):
+        return []
+
+    severity_of_accident = case(
+        [
+            (InvolvedMarkerView.accident_severity < BE_CONST.ACCIDENT_SEVERITY_EASY, 'פצוע קשה'),
```

Comment:
This is not true, since not all involved in a severe accidents are injured. Some are not injured at all.
Also - performing this `InvolvedMarkerView.accident_severity < BE_CONST.ACCIDENT_SEVERITY_EASY` is confusing as well, and we should specify the severities with "in" or "=" and not use < or >.